src/main/java/com/deavensoft/paketomat/courier/CourierModel.java

```diff
@@ -1,16 +1,14 @@
 package com.deavensoft.paketomat.courier;
 
 
-import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Data;
 import lombok.NoArgsConstructor;
-
 import javax.persistence.*;
 
 @NoArgsConstructor
 @Data
 @Entity
-@Table(name = "courier", schema = "public")
+@Table(name = "courier")
 public class CourierModel {
```

Comment:
Why does the class have _Model_ in the name?

Reply:
We have renamed it